### DISCUSSION TITLE
Updated clone images

### DISCUSSION THREAD

**Discussion by freeb1e4me** [nk64/GT4286#2] [Ideas]
I like to suggest clone images of the SD Card showcasing the updates/changes made over time. Keep up the great work

**Comment by nk64** [comment]
I intend to create a command line tool which will hopefully make things one step easier.
But I don't plan to host any of their files (modified or not) unless I am confident that I have the right to do so.

**Reply by mechka99** [reply]
> I intend to create a command line tool which will hopefully make things one step easier. But I don't plan to host any of their files (modified or not) unless I am confident that I have the right to do so.

nk64 you work on this whole thing is so well appreciated :) just waiting on the app to fix the game database etc..

**Reply by horsiezoo** [reply]
Its under releases, can confirm it works
